exercise.wwwapi/Models/Book.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace exercise.wwwapi.Models
+{
+    public class Book
```

Comment:
You can have a model that has the POST and PUT models in. As the classes are the same

Reply:
A little unsure if you mean i could use the same class for post and put or if you want me to make an internal class for books.

Reply:
You have 3 classes `Book`, `BookPost` and `BookPut`  This can all be one class.  We can have a quick 1:1 and run through it?

Reply:
ah yeah, it was planing one using those for the put and post operations, but i used book instead. So i just forgot to delete them.